Sources/SwiftParser/Specifiers.swift

```diff
@@ -615,8 +602,8 @@ extension Parser {
     var asyncKeyword: RawTokenSyntax? = nil
     var unexpectedBeforeThrows: [RawSyntax] = []
     var throwsKeyword: RawTokenSyntax?
-    var thrownError: RawThrownTypeClauseSyntax?
-    var unexpectedAfterThrownError: [RawSyntax] = []
+    var throwsClause: RawThrowsClauseSyntax?
+    var unexpectedAfterThrowsClause: [RawSyntax] = []
```

Comment:
I don’t like having both a `throwsKeyword` and a `throwsClause` here because it’s not clear when we need to include the `throwsKeyword` in the source tree and when the `throwsClause` and thus it’s easy to get source-accuracy wrong.

I would prefer to only have `throwsClause` here and for a `throwsClause` whenever we parse a `throws` keyword.

Reply:
That's fair; I've found a cleaner way to implement it without `throwsKeyword`